application/models/Survey.php

```diff
@@ -550,7 +550,7 @@ public function rules()
             array('running', 'safe', 'on' => 'search'),
             array('expires', 'date','format' => ['yyyy-M-d H:m:s.???','yyyy-M-d H:m:s','yyyy-M-d H:m'],'allowEmpty' => true),
             array('startdate', 'date','format' => ['yyyy-M-d H:m:s.???','yyyy-M-d H:m:s','yyyy-M-d H:m'],'allowEmpty' => true),
```

Comment:
I know you want to fix each issue separately, but here seems there are same issue just 2 lines before.

Can we fix it at same time, it take less time to fix directly it than to report an issue and create another PR …

Reply:
I can test i think , but yes : not really needed.

Reply:
I would make a different PR, maybe a direct commit?

Reply:
Unsure  …

Reply:
Undid the change. That format validator pattern was ok

Reply:
Then array('datecreated', 'default', 'value' => date("Y-m-d H:i:s")), didn't fix issue

Reply:
This was hard to test. The issue is not on the copy process, but on the survey creation one, where the default value is given to the datecreated field

Reply:
Yes, thank you for explanation